haystack/preview/document_stores/memory/document_store.py

```diff
@@ -181,6 +181,7 @@ def write_documents(self, documents: List[Document], policy: DuplicatePolicy = D
             or isinstance(documents, str)
             or any(not isinstance(doc, Document) for doc in documents)
         ):
+            print(documents)
```

Comment:
This is probably left from debugging? :D

Reply:
Uh thank you! :raised_hands:

Reply:
```suggestion
```